src/pipeline/resolver.h

```diff
@@ -351,10 +351,20 @@ namespace librealsense
                     if (!has_wildcards(request)) continue;
                     for (auto candidate : candidates)
                     {
-                        if (match(candidate.get(), request) && !dev->contradicts(candidate.get(), requests))
+                        //if (match(candidate.get(), request) && !dev->contradicts(candidate.get(), requests))
+                        if (match(candidate.get(), request))
                         {
-                            request = to_request(candidate.get());
-                            break;
+                            if (candidate.get()->get_stream_type() == RS2_STREAM_COLOR &&
+                                candidate.get()->get_format() == RS2_FORMAT_RGB8 &&
+                                candidate.get()->get_framerate() == 15)
```

Comment:
Not clear, also - no need to specify the FPS. Probably redundant

Reply:
debugging code - removed